test/stdlib/TestMeasurement.swift

```diff
@@ -147,6 +147,36 @@ class TestMeasurement : TestMeasurementSuper {
         expectTrue(fiveKM < sevenThousandM)
         expectTrue(fiveKM <= fiveThousandM)
     }
+
+    func test_AnyHashableContainingMeasurement() {
+        let values: [Measurement<UnitLength>] = [
+          Measurement(value: 100, unit: UnitLength.meters),
+          Measurement(value: 100, unit: UnitLength.kilometers),
+          Measurement(value: 100, unit: UnitLength.kilometers),
+        ]
+        let anyHashables = values.map(AnyHashable.init)
+        expectEqual(Measurement<UnitLength>.self, type(of: anyHashables[0].base))
+        expectEqual(Measurement<UnitLength>.self, type(of: anyHashables[1].base))
+        expectEqual(Measurement<UnitLength>.self, type(of: anyHashables[2].base))
+        expectNotEqual(anyHashables[0], anyHashables[1])
+        expectEqual(anyHashables[1], anyHashables[2])
+    }
+
+    func test_AnyHashableCreatedFromNSMeasurement() {
+        if #available(iOS 8.0, *) {
```

Comment:
This availability check doesn't make sense to me - NSMeasurement is new in iOS 10 and OS X 10.12.